### DISCUSSION TITLE
Future plans for <DataTable /> component?

### DISCUSSION THREAD

**Discussion by andioneto** [twilio-labs/paste#658] [Office hours] [answered]
Hi folks,
In all of the console product areas that our team builds (e.g. voice, video, sip-trunking, interconnect) we have filterable tables. I see that `DataTable` is on the TBD roadmap and I'm wondering if there has been any early stage planning for that component, or some early design work (e.g. competitive analysis, testing, etc.).

The team would like to build a temporary solution with the components and primitives available from Paste today, but we do not want to cause unnecessary technical debt. I was thinking my teammate Tomeu and I could work on a simple proof of concept and perhaps attend a crit or office hours to get feedback.

As always, thank you for all you do!

**Comment by corinne-nagel** [comment, marked as answer]
Hi Andi! The team just chatted about this, but we're ok with you creating a temporary solution with the knowledge that we'll probably have to rebuild it at a later time when we get around to it. So there's inherently going to be a bit of technical debt but if we all work together on this hopefully it can be something we both benefit from 🙂

**Reply by andioneto** [reply]
Ok awesome! Would it be helpful to have a small kick off for this? Otherwise, we can start with simple composed presentational components, go to crit for feedback, etc. Let me know what kind of collaboration works best for the team 😺

**Reply by corinne-nagel** [reply]
I think since you're building it with Paste components and primitives that you're already off to a great start; let's keep checking in in office hours/crits on a regular cadence. But if you have any bigger upfront questions on functionality/how it's built we could also do a kickoff, however you need our support 👍

**Comment by andioneto** [comment]
@tomtobac
